extensions-contrib/ambari-metrics-emitter/pom.xml

```diff
@@ -48,6 +48,18 @@
       <type>test-jar</type>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
what is this change about, was this relying on curator providing it?

Reply:
Yes. curator 4.1.0 was providing both org.codehaus.jackson:jackson-mapper-asl:jar:1.9.13 and org.codehaus.jackson:jackson-core-asl:jar:1.9.13. Curator 4.3.0 no longer provides those which then causes java.lang.NoClassDefFoundError: org/codehaus/jackson/map/AnnotationIntrospector.

Reply:
Hence, explicitly adding dependecy on those org.codehaus.jackson:jackson-mapper-asl:jar and org.codehaus.jackson:jackson-core-asl:jar

Reply:
Actually I think they can be scope test

Reply:
Changed to scope test.

Reply:
This was the dependency tree before the change in this PR
[INFO] |  +- org.apache.curator:curator-x-discovery:jar:4.1.0:test
[INFO] |  |  \- org.codehaus.jackson:jackson-mapper-asl:jar:1.9.13:test
[INFO] |  |     \- org.codehaus.jackson:jackson-core-asl:jar:1.9.13:test

Reply:
If jackson-mapper-asl:1.9.13 is no longer in production code, can you check if its security vulnerability suppression can be removed? https://github.com/apache/druid/blob/master/owasp-dependency-check-suppressions.xml#L125

You can run the security vulnerability scan by running: `mvn dependency-check:check`

Reply:
If you skip druid console, do all the other modules pass? If so, then the suppression for jackson-mapper-asl can be removed. We'll need to fix the other issues in different PRs.

Reply:
No. Other modules such as druid-orc-extensions and druid-hdfs-storage fails. This is because jackson-mapper-asl-1.9.13.jar is pulled in by org.apache.hadoop:hadoop-common:jar:2.8.5